pcdsdevices/mv_interface.py

```diff
@@ -6,6 +6,8 @@
 import logging
 import numbers
 import signal
+import pylab
+from pylab import plot, ginput, show, axis
```

Comment:
You should pick one of these schemas and stick to it to avoid a messy import block. We don't need to import `plot` from `pylab` and then later call `pylab.plot` instead of just `plot`. One or the other would make the code more readable for all the `pylab` functions.